src/main/java/org/fundacionjala/sfdc/pageobjects/Opportunities/SFOEdit.java

```diff
@@ -0,0 +1,125 @@
+package org.fundacionjala.sfdc.pageobjects.Opportunities;
+
+import org.fundacionjala.sfdc.commons.DriverManager;
+import org.fundacionjala.sfdc.pageobjects.SalesForceConnection;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.How;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.Select;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.util.List;
+
+/**
+ * SFOEdit.java
+ * Class that represents the page where an oppotunity could be modified.
+ */
+public class SFOEdit extends SalesForceConnection {
+    private WebDriverWait wait;
+
+    @FindBy(how = How.ID, using = "opp3")
+    private WebElement opportunityName;
+
+    @FindBy(how = How.ID, using = "opp9")
+    private WebElement opportunityCloseDate;
+
+    @FindBy(how = How.ID, using = "opp11")
+    private WebElement opportunityStage;
+
+    @FindBy(how = How.CSS, using = "#topButtonRow > input:nth-child(1)")
+    private WebElement saveOpportunityButton;
+
+    @FindBy(how = How.ID, using = "AllTab_Tab")
+    private WebElement pagesButton;
+
+    @FindBy(how = How.LINK_TEXT, using = "Opportunities")
+    private WebElement opportunitiesPage;
+
+    /**
+     * Constructor.
+     */
+    public SFOEdit() {
+        super();
+        PageFactory.initElements(this.webDriver, this);
+    }
+
+    /**
+     * Method to get "save" button.
+     *
+     * @return web element save button.
+     */
+    public WebElement getSaveOpportunityButton() {
+        wait = DriverManager.getInstance().getWaiter();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Change this instance-reference to a static reference. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2209)